pkg/apis/extensions/v1beta1/types.go

```diff
@@ -215,6 +215,10 @@ type DeploymentSpec struct {
 	UniqueLabelKey *string `json:"uniqueLabelKey,omitempty"`
 }
 
+const (
+	DefaultDeploymentUniqueLabelKey string = "deployment.kubernetes.io/podTemplateHash"
```

Comment:
Add this godoc:
```
// DefaultDeploymentUniqueLabelKey is the default key of the selector that is added
// to existing RCs (and label key that is added to its pods) to prevent the existing RCs
// to select new pods (and old pods being selected by new RC).
```